x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/process/crossvalidation/SingleClassReservoirCrossValidationSplitter.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.ml.dataframe.process.crossvalidation;
+
+import java.util.List;
+
+public class SingleClassReservoirCrossValidationSplitter extends AbstractReservoirCrossValidationSplitter {
+
+    private final SampleInfo sampleInfo;
+
+    SingleClassReservoirCrossValidationSplitter(List<String> fieldNames, String dependentVariable, double trainingPercent,
+                                                long randomizeSeed, long trainingCardinality) {
```

Comment:
I always double take when I read cardinality in the `SampleInfo` and here in this class. 

The scope what what cardinality we are checking is unclear. Usually when I read cardinality I associate it with the  number of classes. But in this case, cardinality is the count of documents in our specific class

Reply:
Yeah, I see what you mean. Shall I rename to `docCount` or something?